vizro-ai/examples/dashboard_ui/actions.py

```diff
@@ -13,16 +15,50 @@
 from vizro.models.types import capture
 from vizro_ai import VizroAI
 
+try:
+    from langchain_anthropic import ChatAnthropic
+except ImportError:
+    ChatAnthropic = None
+
+try:
+    from langchain_mistralai import ChatMistralAI
+except ImportError:
+    ChatMistralAI = None
+
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)  # TODO: remove manual setting and make centrally controlled
 
-SUPPORTED_VENDORS = {"OpenAI": ChatOpenAI}
+SUPPORTED_VENDORS = {"OpenAI": ChatOpenAI, "Anthropic": ChatAnthropic, "Mistral": ChatMistralAI}
+
+SUPPORTED_MODELS = {
+    "OpenAI": [
+        "gpt-4o-mini",
+        "gpt-4o",
+        "gpt-4",
+        "gpt-4-turbo",
+        "gpt-3.5-turbo",
+    ],
+    "Anthropic": [
+        "claude-3-5-sonnet-20240620",
+        "claude-3-opus-20240229",
+        "claude-3-sonnet-20240229",
+        "claude-3-haiku-20240307",
+    ],
+    "Mistral": ["mistral-large-latest", "open-mistral-nemo", "codestral-latest"],
```

Comment:
Shall we consider removing mistral models for now, because we don't have the API key internally. @maxschulz-COL

Reply:
For me it very much worked!

Reply:
![image](https://github.com/user-attachments/assets/a5d17834-65d5-4562-ac7c-3e90a418d00d)

Reply:
Did you host the model locally? If anyone of you tested it and it works fine, then yes let's keep mistral 👍

Reply:
No, we do have the API key internally! At least I do 😂